spec/tags/core/file/stat/inspect_tags.txt

```diff
@@ -0,0 +1,2 @@
+fails(darwin):File::Stat#inspect produces a nicely formatted description of a File::Stat object
+
```

Comment:
There seems to be an extra newline here.
What's the failure actually?

Reply:
Missing `File::Stat#birthtime`, IIRC.